### PR TITLE
Landing Video

### PR DESCRIPTION
**Pull request by harshvitra**
Status:

- [x]  Prevent video from loop
- [x] Fix video not playing on the first load on many mobile devices.

### CODE REVIEW

File: src/features/common/LandingVideo/index.tsx [hunk 1]
```diff
@@ -27,34 +27,34 @@ function VideoContainer({ setshowVideo }: Props): ReactElement {
       switch (localStorage.getItem('language')) {
         case 'de':
           setvideoURL(
-            `${process.env.CDN_URL}/landing-vid/planet/en-intro-mobile-planet.mp4`
+            `https://a.plant-for-the-planet.org/wp-content/videos/en-intro-mobile-planet.mp4`
```
Comment (norbertschuler) [inline src/features/common/LandingVideo/index.tsx:30]: How does this fit with https://github.com/Plant-for-the-Planet-org/plant-for-the-planet-org.github.io/pull/22/files?

Comment (sagararyal) [reply src/features/common/LandingVideo/index.tsx:30]: @norbertschuler file size limits, still testing.

Comment (sagararyal) [reply src/features/common/LandingVideo/index.tsx:30]: Problem is mainly Safari iOS, video does not load automatically.

**Comment by sagararyal** [comment]
- [ ] would be lovely to have support for `canplaythrough` HTML event and disable video where the response is false.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/canplaythrough_event

- [ ] Based on user agent googlebot; etc mark the video disabled.

**Pull request merged**